### PR TITLE
chore: run CI tests on more node versions

### PR DESCRIPTION
**Pull request by wa0x6e**
Run the github action tests on more node versions.

It's currently only ran on node 14, which is dead for many years.

Adding 16 (which is EOL but is still used by many of our services), 18 (current) and 20.

This will avoid scenario like https://github.com/snapshot-labs/snapshot.js/pull/904 which fails on node 18

### CODE REVIEW

**Comment by ChaituVR** [comment]
But this will increase number of actions 
image

we may hit concurrent runs soon, which may cause some delay to merge things https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#about-billing-for-github-actions

**Comment by wa0x6e** [comment]
We can 2:
- the extreme: so 14 and 20
- the most used: 16 and 18

wdyt?

**Comment by ChaituVR** [comment]
that would be perfect 👌

**Comment by wa0x6e** [comment]
> that would be perfect 👌

Which one?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: ['14', '16', '18', '20']
```
Comment (ChaituVR) [inline .github/workflows/test.yml:11]: I think extremes we want to support 
```suggestion
        node-version: ['14', '20']
```

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**